### PR TITLE
chore: Refactor Test Clients to use "resources" directory

### LINKED ISSUE #3361: Test clients "resource" dir is misnamed
### Problem

The standard naming convention for maven/gradle is `src/main/resources`. test-clients is missing the "s" at the end.

### Solution

Rename test-clients/src/main/resource to test-clients/src/main/resources. Update all places that are hardcoded with "resource" in the path name (there are a few places).

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by thomas-swirlds-labs**
**Description**:

- Rename `test-clients/src/main/resource` to `test-clients/src/main/resources`. 
- Update all places that are hardcoded with "resource" in the path name (there are a few places).

**Related issue(s)**:

Fixes #3361 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by david-bakin-sl — COMMENTED** [review]
Don't forget to update `test-clients/build.gradle.kts` to remove the exception for `src/main/resource`!  [Here](https://github.com/hashgraph/hedera-services/blob/8011da635373a354ca3b5a81af61da0e615fdb8a/hedera-node/test-clients/build.gradle.kts#L40)

**Comment by thomas-swirlds-labs** [comment]
> Don't forget to update `test-clients/build.gradle.kts` to remove the exception for `src/main/resource`! [Here](https://github.com/hashgraph/hedera-services/blob/8011da635373a354ca3b5a81af61da0e615fdb8a/hedera-node/test-clients/build.gradle.kts#L40)

Good catch! Thanks @david-bakin-sl. Updated 👍

**Review by david-bakin-sl — APPROVED** [review]
LGTM and bdds pass with this rename.

**Review by mhess-swl — DISMISSED** [review]

**Review by iwsimon — DISMISSED** [review]
LGTM. Thanks @thomas-swirlds-labs

File: .gitignore [hunk 1]
```diff
@@ -28,7 +28,7 @@ buildNumber.properties
 
 #Mono Project Files
 *.pidb
-*.resources
+hedera-node/hedera-mono-service/src/test/resources/
```
Comment (nathanklick) [inline .gitignore:31]: Why are we excluding the mono svc resources path and removing the `*.resources` file exclusions. This line has nothing to do with the `hedera-mono-service` module and is related to the MonoDevelop IDE exclusions. 

These changes should be reverted.

**Review by nathanklick — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (thomas-swirlds-labs) [reply .gitignore]: Reverted now 👍 Thanks @nathanklick !

**Review by nathanklick — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]
LGTM

**Pull request merged**